app/build.gradle

```diff
@@ -167,7 +167,7 @@ dependencies {
     }
 
     implementation "com.github.shts:TriangleLabelView:$triangleLabelViewVersion"
-    implementation 'com.leinardi.android:speed-dial:3.1.0'
+    implementation 'com.leinardi.android:speed-dial:3.0.0'
```

Comment:
Alternatively you could force the version of material components to 1.0.0, this way you'll still be able to use speed-dial 3.1.0 without adding the 2k+ methods:
```
configurations.all {
    resolutionStrategy {
        force 'com.google.android.material:material:1.0.0'
    }
}
```
More here: https://github.com/leinardi/FloatingActionButtonSpeedDial/issues/136#issuecomment-536520999